website/docs/docs/use-dbt-semantic-layer/sl-cache.md

```diff
@@ -44,6 +44,16 @@ Different data platforms might have different caching layers and cache invalidat
 
 Declarative caching enables you to pre-warm the cache using [saved queries](/docs/build/saved-queries) by setting the cache config to `true` in your `saved_queries` settings. This is useful for optimizing performance for key dashboards or common ad-hoc query requests. For configuration details, refer to [Declarative caching setup](#declarative-caching-setup).
 
+<details>
+
+<summary> 📹 Check out this video demo to see how Declarative caching works!</summary>
```

Comment:
```suggestion
<summary> 📹 Check out this video demo to see how declarative caching works!</summary>
```

Reply:
yea i was back and forth w it - the name of the feature is 'Declarative' so that's why i had it like this. but all good!